RHDEVDOCS-2335: Added a procedure to monitor application and image vulnerabilities metrics (#48153)
openshift/openshift-docs: modules/odc-image-vulnerabilities-breakdown.adoc

// Module included in the following assemblies:
//
// * applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc

:_content-type: CONCEPT
[id="odc-image-vulnerabilities-breakdown_{context}"]
= Image vulnerabilities breakdown

> In the developer perspective, the project dashboard shows the *Image Vulnerabilities* link in the *Status* section. Using this link, you can view the *Image Vulnerabilities breakdown* pop-up that includes details regarding vulnerable container images and fixable container images. The severity level of vulnerability is highlighted using the following color icons:

Reviewer: ```suggestion
In the developer perspective, the project dashboard shows the *Image Vulnerabilities* link in the *Status* section. Using this link, you can view the *Image Vulnerabilities breakdown* window, which includes details regarding vulnerable container images and fixable container images. The icon color indicates severity:
```
- IBM SG: pop up adjective Do not use. Use "pop-up", but only if you must specify the type of menu, list, or window.
- Use non-restrictive `, which` instead of `that`.

> * Red: Indicates the vulnerability is of high priority for container image functioning and needs an immediate fix
> * Orange: Indicates the vulnerability is of medium priority and can be fixed after high priority vulnerabilities are fixed
> * Yellow: Indicates the vulnerability is of low priority and can be fixed after high and medium priority vulnerabilities are fixed

Reviewer: ```suggestion
* Red: High priority. Fix immediately.
* Orange: Medium priority. Can be fixed after high-priority vulnerabilities.
* Yellow: Low priority. Can be fixed after medium priority vulnerabilities.
```
Optional: Minimalism. I think you can remove redundant words and increase clarity.
